.gitmodules

```diff
@@ -17,7 +17,7 @@
 [submodule "GPLOT"]
     path = sorc/hafs_graphics.fd/hrd_gplot
     url = https://github.com/hafs-community/GPLOT.git
-    branch = support/HAFS
+    branch = release/0.4.0
```

Comment:
@ghassan-alaka, since the HAFS develop branch typically has its submodules point to the support/HAFS branch or the corresponding authorized main or develop branch, could you please merge the release/0.4.0 branch into the support/HAFS branch of https://github.com/hafs-community/GPLOT.git. After that, you can revert this change to point to the support/HAFS branch here. Thanks!

Reply:
OK sounds good @BinLiu-NOAA . I just updated the `support/HAFS` branch within the hafs-community/GPLOT repo.